.github/workflows/publish-deb-package.yml

```diff
@@ -0,0 +1,91 @@
+        type: string
+      USE_TEST_REPO:
+        description: 'Use test repo for publishing'
+        required: true
+        type: string
+      PACKAGE_VERSION:
+        description: 'Version of the package e.g. 5.1.1, 5.1.1-1, defaults to HZ_VERSION'
+        type: string
+
+env:
+  JFROG_TOKEN: ${{ secrets.JFROG_TOKEN }}
+
+concurrency:
+  group: 'deb-${{ github.job }}-${{ inputs.HZ_VERSION }}-${{ inputs.HZ_DISTRIBUTION}}'
+
+jobs:
+  deb:
+    runs-on: ubuntu-latest
+    env:
+      HZ_VERSION: ${{ inputs.HZ_VERSION }}
+      PACKAGE_VERSION: ${{ inputs.PACKAGE_VERSION || inputs.HZ_VERSION }}
+      HZ_DISTRIBUTION: ${{ inputs.HZ_DISTRIBUTION }}
+      PUBLISH: "true"
+      USE_TEST_REPO: ${{ inputs.USE_TEST_REPO }}
+
+    steps:
+      - name: Checkout hazelcast-packaging repo
+        uses: actions/checkout@v4
+
+      - name: Load env vars from .env file
+        run: cat .env >> $GITHUB_ENV
+
+      - name: Setup JDK
+        uses: actions/setup-java@v4
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+
+      - name: Download the distribution tar.gz file
+        run: |
+          mvn --batch-mode dependency:copy -Dartifact=com.hazelcast:${HZ_DISTRIBUTION}-distribution:${HZ_VERSION}:tar.gz -DoutputDirectory=./ -Dmdep.useBaseVersion=true
+
+      - name: Create & Upload DEB package
+        run: |
+          ./build-hazelcast-deb-package.sh
+
+      - name: Calculate Debian Repository Metadata
+        run: |
+          source common.sh
+
+          curl --fail-with-body --retry 3 --retry-delay 10 -H "Authorization: Bearer ${{ env.JFROG_TOKEN }}" \
+            -X POST "https://repository.hazelcast.com/api/deb/reindex/${DEBIAN_REPO}"
+
+      - name: Install Hazelcast from deb
+        env:
+          HZ_LICENSEKEY: ${{ secrets.HZ_LICENSEKEY }}
+        run: |
+          source ./common.sh
+          wget -qO - https://repository.hazelcast.com/api/gpg/key/public | gpg --dearmor | sudo tee /usr/share/keyrings/hazelcast-archive-keyring.gpg > /dev/null
+          echo "deb [signed-by=/usr/share/keyrings/hazelcast-archive-keyring.gpg] ${DEBIAN_REPO_BASE_URL} ${PACKAGE_REPO} main" | sudo tee -a /etc/apt/sources.list
+          sudo apt update && sudo apt install ${{ env.HZ_DISTRIBUTION}}=${HZ_VERSION}
+          HAZELCAST_CONFIG="$(pwd)/config/integration-test-hazelcast.yaml" hz-start > hz.log 2>&1 &
+
+      - name: Check Hazelcast health
+        run: |
+          ./check-hazelcast-health.sh
+
+      - name: Uninstall Hazelcast from deb
+        run: |
+          source ./common.sh
+          sudo apt remove ${{ env.HZ_DISTRIBUTION}}
+
+      - name: Remove deb package from test repo
+        if: env.USE_TEST_REPO == 'true'
+        run: |
+          source ./common.sh
+          curl -H "Authorization: Bearer ${{ env.JFROG_TOKEN }}" \
+            -X DELETE \
+            "$DEBIAN_REPO_BASE_URL/${HZ_DISTRIBUTION}-${DEB_PACKAGE_VERSION}-all.deb"
```

Comment:
Is there a problem with the line ending of the file?

Reply:
Fixed: https://github.com/hazelcast/hazelcast-packaging/pull/204/commits/b76dcfb7a2fc52b96aaaf7ebd3c3198dafa258ca